pkg/v1/tkg/tkgctl/init.go

```diff
@@ -276,9 +277,9 @@ func (t *tkgctl) configureInitManagementClusterOptionsFromConfigFile(iro *InitRe
 
 	// set BuildEdition from config variable
 	if iro.Edition == "" {
-		edition, err := t.TKGConfigReaderWriter().Get(constants.ConfigVariableBuildEdition)
+		cfg, err := config.GetClientConfig()
```

Comment:
Same as above. (We should try not to get an edition from config file at this place as this is part of the library code and we might want the users to provide an edition with the `BUILD_EDITION` config variable)